spec/requests/v2/create_response_spec.rb

```diff
@@ -14,7 +14,7 @@
 
     shared_context 'with staging folder visibility' do
```

Comment:
Re structured to keep atos examples separate (ready for removal in march time)

Added examples to test for correct routing of data